### PR TITLE
KevS-SK1289-KFT0009249-Update-bdd-overquota-web-hosting+TRANSLATIONS

### PR DESCRIPTION
**Pull request by Kazarus4**
Adding new guide about db duplication (link) => two times on the guide.

WARNING ===>>> DON'T MERGE THIS PR WHILE JIRA SK1271 ISN'T DONE TOO.

@BrenKt can you already do a peer rv plz. This to prepare this update quickly and merge quickly after the merging of the new feature's guide (JIRA SK1271).

### CODE REVIEW

File: pages/web_cloud/web_hosting/sql_overquota_database/guide.fr-fr.md [hunk 1]
```diff
@@ -124,7 +124,9 @@ Consultez notre offre de bases de données [Web Cloud Databasess](https://www.ov
 
 Nous recommandons cette offre pour les bases de données volumineuses.
 
-Suivez ensuite nos guides pour déplacer le contenu de votre ancienne base de données vers la nouvelle :
+Vous pouvez dupliquer rapidement le contenu de votre base de données OVHcloud dans une autre base de données OVHcloud grâce à une fonctionnalité présente dans votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr). Retrouvez plus de détails dans notre guide « [Dupliquer le contenu d'une base de données dans une autre](/pages/web_cloud/web_hosting/copy_database) ».
```
Comment (BrenKt) [inline pages/web_cloud/web_hosting/sql_overquota_database/guide.fr-fr.md]:
```suggestion
Il est possible de dupliquer le contenu de votre base de données OVHcloud directement vers une autre de vos bases de données OVHcloud grâce à une fonctionnalité présente dans votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr). Pour cela, consultez notre guide « [Dupliquer le contenu d'une base de données dans une autre](/pages/web_cloud/web_hosting/copy_database) ».
```

File: pages/web_cloud/web_hosting/sql_overquota_database/guide.fr-fr.md [hunk 2]
```diff
@@ -124,7 +124,9 @@ Consultez notre offre de bases de données [Web Cloud Databasess](https://www.ov
...
+
+Sinon, vous pouvez suivre nos guides pour déplacer le contenu de votre ancienne base de données vers la nouvelle manuellement :
```
Comment (BrenKt) [inline pages/web_cloud/web_hosting/sql_overquota_database/guide.fr-fr.md]:
```suggestion
Dans le cas d'une migration vers une base de données externe aux offres [Start SQL](https://www.ovhcloud.com/fr/web-hosting/options/start-sql/) et [Web Cloud Databases](https://www.ovhcloud.com/fr/web-cloud/databases/), vous pouvez déplacer manuellement le contenu de votre ancienne base de données vers une nouvelle à l'aide de nos guides:
```

**Comment by Kazarus4** [comment]
Need DE and ENGB proof too by @tcpdumpfbacke

**Comment by Kazarus4** [comment]
JIRA SK1271 DONE since 22/11/2023

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**